### PR TITLE
[Doc] Fix some fault example code of loading data using Spark connector

### PR DESCRIPTION
**Pull request by Czkonverse**
Why I'm doing:
1. The back slash in create schema for df is redundant.
2. The code was error because of the  original data format with "\\" and annotation.

What I'm doing:
1. Delete the redundant back slash.
2. Wrap the PySpark code with bracket, so that the code can be used directly with annotation in the original place.

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
- [x] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.2
  - [ ] 3.1
  - [x] 3.0
  - [ ] 2.5

### CODE REVIEW

**Review by DanRoscigno — APPROVED** [review]
Thanks so much @Czkonverse !

**Review by imay — APPROVED** [review]

**Comment by imay** [comment]
@Czkonverse Thank you for the contribution, could you sign the CLA, otherwise we can not accept it. You need to commit this by `git commit -s ` to sign-off this commit. Then we can accept it and you will become a contributor to StarRocks.

File: docs/loading/Spark-connector-starrocks.md [hunk 1]
```diff
@@ -303,31 +303,34 @@ Construct a streaming read of data from a CSV file and load data into the StarRo
         .getOrCreate()
```
Comment (hellolilyliuyi) [inline docs/loading/Spark-connector-starrocks.md:303]: Actually, I verified all the examples when I wrote this doc. Are you sure that an error is returned when the following codes are executed? The use of backslashes \ is for line continuation, enhancing code readability by breaking long lines into more readable multiple lines. In this context, the backslashes have no impact on the functionality of the code.

Comment (Czkonverse) [reply docs/loading/Spark-connector-starrocks.md:303]: Yes, I'm sure, please just copy the original code to your IDE and see the prompts of the IDE. Especially the last right bracket, no matched left bracket before.

**Comment by imay** [comment]
@mergify backport branch-3.1 branch-3.2

**Pull request merged**